src/Cnab.Bundle/Bundle.cs

```diff
@@ -33,11 +34,12 @@ public class Bundle
         [JsonProperty("actions")]
         public Dictionary<string, Action> Actions { get; set; }
 
-        public static Bundle LoadUnsigned(string fileName)
+        public static async Task<Bundle> LoadUnsignedAsync(string fileName)
```

Comment:
Do we want to have both sync and async methods?
What is the best practice in this case?

Reply:
Async methods everywhere, unless there's very clear reasoning to have it sync, or the super edge case where you want both.
Ref: https://blogs.msdn.microsoft.com/pfxteam/2012/04/13/should-i-expose-synchronous-wrappers-for-asynchronous-methods/

I'll fix the README as well, missed that.

Reply:
The slight difference here is that we would not return `.Result` from an async method, but rely on `StreamReader` and `JsonConvert` to do their operations synchronously.

That being said, I'm having an asynchronous API for now and come back later and see if a synchronous one is actually needed.
cc @itowlson, who I think had his fair share of `sync` vs. `async` kerfuffles.

Reply:
10 Points for `kerfuffles`.